.version

```diff
@@ -1 +1 @@
-0.9.9
+0.9.10-debug4
```

Comment:
same comment I made on #119 that these versions should be updated if we're actually going to merge this PR. Otherwise, it seems it will be handled by #137